pkg/autoupdate/findnew_test.go

```diff
@@ -415,6 +412,7 @@ func TestCheckExecutableTruncated(t *testing.T) {
 
 func TestBuildTimestamp(t *testing.T) {
 	t.Parallel()
+	t.Skip("FIXME: Windows")
```

Comment:
could we wrap this in a `if runtime.GOOS == "windows ` ?

Reply:
OMG. That's a great catch. Thanks!